# HG changeset patch
# User andrew
# Date 1549859737 0
#      Mon Feb 11 04:35:37 2019 +0000
# Node ID 9087f96c9d956e0141741639445ebcb8bb20e3e6
# Parent  9ecf75ce7c7056614f22ae32745c1f06fa312482
8217753, PR3685: Enable HotSpot builds on 5.x Linux kernels
Summary: Remove OS version check on Linux as very unlikely a 2.2.x kernel is still in use

diff --git openjdk.orig/hotspot/make/linux/Makefile openjdk/hotspot/make/linux/Makefile
--- openjdk.orig/hotspot/make/linux/Makefile
+++ openjdk/hotspot/make/linux/Makefile
@@ -227,20 +227,7 @@
 	@echo "  $(TARGETS_SHARK)"
 	@echo "  $(TARGETS_MINIMAL1)"
 
-checks: check_os_version check_j2se_version
-
-# We do not want people accidentally building on old systems (e.g. Linux 2.2.x,
-# Solaris 2.5.1, 2.6).
-# Disable this check by setting DISABLE_HOTSPOT_OS_VERSION_CHECK=ok.
-
-SUPPORTED_OS_VERSION = 2.4% 2.5% 2.6% 3% 4%
-OS_VERSION := $(shell uname -r)
-EMPTY_IF_NOT_SUPPORTED = $(filter $(SUPPORTED_OS_VERSION),$(OS_VERSION))
-
-check_os_version:
-ifeq ($(DISABLE_HOTSPOT_OS_VERSION_CHECK)$(EMPTY_IF_NOT_SUPPORTED),)
-	$(QUIETLY) >&2 echo "*** This OS is not supported:" `uname -a`; exit 1;
-endif
+checks: check_j2se_version
 
 # jvmti.make requires XSLT (J2SE 1.4.x or newer):
 XSLT_CHECK	= $(REMOTE) $(RUN.JAVAP) javax.xml.transform.TransformerFactory
